swarm/swarm.go

```diff
@@ -524,8 +524,33 @@ func (self *Swarm) SetChequebook(ctx context.Context) error {
 type Info struct {
 	*api.Config
 	*chequebook.Params
+	StoreStats *LocalStoreStats
 }
 
 func (self *Info) Info() *Info {
+	self.StoreStats.Info()
 	return self
 }
+
+type LocalStoreStats struct {
+	getters map[string]func() interface{}
+
+	DbCapacity   uint64
```

Comment:
DbCapacity is already exposed in `Info` as `api.Config` has the value from configuration `DbCapacity`, which is just assigned to LDBStore, not changed on initialization or in runtime. This information is redundant in the current code.